board/common/qemu/qemu.sh

```diff
@@ -192,6 +192,38 @@ net_args()
     fi
 }
 
+# Vital Board Data, part of Vital Product data
+vbd_args()
+{
+    onieprom="$(dirname "$CONFIG_QEMU_ROOTFS")/onieprom"
+    vbd_file="$(dirname "$CONFIG_QEMU_ROOTFS")/vbd"
+
+    # This is you QEMU factory/default password:
+    pwhash=$(echo -n "admin" | mkpasswd -s -m sha256crypt)
+
+    # NOTE: This format will change!
+    json_data=$(cat <<EOF
+{
+  "manufacture-date": "$(date +"%d/%m/%Y %H:%M:%S")",
+  "vendor-extension": [
+    [
+      65536,
```

Comment:
```suggestion
      61046,
```

This should use Kernelkit's IANA Enterprise Number